src/main/java/seedu/address/logic/parser/EditGuestCommandParser.java

```diff
@@ -45,6 +50,10 @@ public EditGuestCommand parse(String args) throws ParseException {
             throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, EditGuestCommand.MESSAGE_USAGE), pe);
         }
 
+        if (index.getOneBased() > this.model.getFilteredGuestList().size()) {
+            throw new ParseException(Messages.MESSAGE_INVALID_GUEST_DISPLAYED_INDEX);
+        }
```

Comment:
Any reason why this check has to be done in the parser when the same check is already at EditGuestCommand?

Likewise for delete